src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: bold;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.search__bar {
+  box-sizing: border-box;
+  width: 100%;
+  background: #ffff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-weight: bold;
+  color: #3d4e61;
+}
+
+.search__bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search__bar:focus {
+  background: linear-gradient(180deg, #ffff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  outline: none;
+}
+
+.search__bar::placeholder {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search__bar--big {
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+}
```

Comment:
checklist point 2
if you have two or more similar elements with portions of similar styles with different values - use one of the elements as the basic case, and override necessary styles for other cases. Explanation: The point is not in the names of the classes, the point is: when there are several similar elements, ex., 2 inputs, for one we can give a class input, for example, and for the second - input input--small. We write all the styles for .input, but for .input-small we write only those styles that differ in design, and we need this second input to look a little different. Element with class .input without extra classes should also look like a full-fledged styled element.